torch/csrc/autograd/utils/python_arg_parsing.h

```diff
@@ -7,25 +7,26 @@
 
 namespace torch { namespace autograd { namespace utils {
 
-inline std::tuple<at::optional<at::Device>, at::optional<at::ScalarType>, bool>
+inline std::tuple<at::optional<at::Device>, at::optional<at::ScalarType>, bool, bool>
 parse_to_conversion(PyObject *args, PyObject *kwargs) {
   static PythonArgParser parser({
-    "to(Device device=None, ScalarType dtype=None, bool non_blocking=False)",
-    "to(ScalarType dtype, bool non_blocking=False)",
-    "to(Tensor tensor, bool non_blocking=False)",
+    "to(Device device=None, ScalarType dtype=None, bool non_blocking=False, bool copy=False)",
```

Comment:
Actually. This is changing the module.to behavior! Now you are accepting an extra optional `copy` keyword. Could you fix that? Sorry for not catching it earlier.

Reply:
Oh oh. Yes. Sorry, I didn't think about the modules...